transitions/generate_patient_id_on_patients.js

```diff
@@ -0,0 +1,11 @@
+var transitionUtils = require('./utils');
+
+module.exports = {
+  filter: function(doc) {
+    return doc.type === 'person' &&
```

Comment:
I like the idea of using the `patient_id` as a "short ID" or something that can be used for any person to do various things, that makes perfect sense.  The only problem is generating them is a little expensive, requires querying each time.  Have we discussed potentially changing the IDs to `person-$id` I feel the way we are dealing with `patient_id` currently is pretty brittle.  Adding `patient_id` to every person by default makes it more brittle unfortunately. Thinking...